### PR TITLE
Asset editor improvements

### PR DESCRIPTION
**Pull request by tomekpaszek**
- Added some UI tests
- Added support for "Use in control schemes"
- Improved filtering and searching
- Made newly created item automatically selected
- Various UX improvements

### CODE REVIEW

File: Packages/com.unity.inputsystem/Tests/InputSystem/Editor/AssetEditorTests.cs [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    public void Cleanup()
+    {
+        foreach (var w in Resources.FindObjectsOfTypeAll<AssetInspectorWindow>())
+        {
+            w.CloseWithoutSaving();
+        }
+    }
+
+    [Test]
+    public void FilteringByName()
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/Tests/InputSystem/Editor/AssetEditorTests.cs:30]: No `XXXTest` or `XXXWorks` or `XXX` tests. Every test should clearly specify a behavior in a readable requirement. `CanDoThis` or `WhenDoingThis_ThisHappens` or `CannotDoThat`. Stuff like that.

File: Packages/com.unity.inputsystem/Tests/InputSystem/Editor/AssetEditorTests.cs [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        yield return null;
+
+        // Is new composite selected
+        var selectedRow = (ActionTreeItem)assetWindow.m_ActionsTree.GetSelectedRow();
+        Assert.That(selectedRow.displayName, Is.EqualTo("action"));
+    }
+
+    [UnityTest]
+    [Ignore("For some reason it's impossible to focus the tree view from the test")]
+    public IEnumerator CanCopyAndPaste()
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/Tests/InputSystem/Editor/AssetEditorTests.cs:168]: Mark `TODO_`.

File: Packages/com.unity.inputsystem/Tests/InputSystem/Editor/EditorTests.asmdef [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    ],
+    "optionalUnityReferences": [
+        "TestAssemblies"
+    ],
+    "includePlatforms": [
+        "Editor"
+    ],
+    "excludePlatforms": [],
+    "allowUnsafeCode": false
+}
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/Tests/InputSystem/Editor/EditorTests.asmdef:14]: Let's keep it all together. We have all the system code in one assembly, let's keep all the test code in one assembly, too.

Comment (tomekpaszek) [reply Packages/com.unity.inputsystem/Tests/InputSystem/Editor/EditorTests.asmdef:14]: It's edit mode tests. I would find it weird to run those in play mode test but I can move them if you'd prefer that.

**Pull request merged**